libc/docs/gpu/building.rst

```diff
@@ -0,0 +1,243 @@
+.. _libc_gpu_building:
+
+======================
+Building libs for GPUs
+======================
+
+.. contents:: Table of Contents
+  :depth: 4
+  :local:
+
+Building the GPU C library
+========================
```

Comment:
I think this line needs to match the length of the line above for the formatting to work

Reply:
Updated, thanks.

Reply:
/me checks notes

```
    To build libc docs:
    - Configure with `-DLLVM_ENABLE_SPHINX=ON -DLIBC_INCLUDE_DOCS=ON`
    - Build with `ninja docs-libc-html`
```
then you can verify the output locally.

Reply:
I think this shows up as a warning but will usually do the expected thing. At least I didn't notice anything  different in the output HTML. But we try to build all Sphinx with warnings as errors on, but I usually turn it off since there's always one project with an error in it.